cmd/autoheal/active_jobs_worker.go

```diff
@@ -21,15 +21,15 @@ import (
 	"k8s.io/apimachinery/pkg/util/runtime"
 )
 
-func (h *Healer) runActiveJobsWorker() {
+func (r AWXJobRunner) runActiveJobsWorker() {
```

Comment:
`r` should be a pointer, otherwise you will be copying the contents of the struct each time you call this method (and any other method that has the struct as receiver):

```go
func (r *AWXJobRunner) runActiveJobsWorker() {
  ...
}
```